output_channels: 1
feature_extractor_name: "resnet50"
confidence_thresh: 0.3
use_bgr: True
use_bgr2gray: False